pkg/util/template_util.go

```diff
@@ -0,0 +1,104 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
```

Comment:
this is a dupe of https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/pkg/common/template_util.go